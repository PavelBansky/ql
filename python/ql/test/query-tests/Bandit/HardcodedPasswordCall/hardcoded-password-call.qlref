Bandit/hardcoded-password-call.ql