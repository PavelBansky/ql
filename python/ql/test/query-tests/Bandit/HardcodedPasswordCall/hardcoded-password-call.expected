| Test.py:22:1:22:25 | doLogin() | Possible hardcoded password. |
